pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1013,7 +1041,11 @@ bool GSQLBackend::get(DNSResourceRecord &r)
     } catch (SSqlException &e) {
       throw PDNSException("GSQLBackend get: "+e.txtReason());
     }
-    extractRecord(row, r);
+    try {
+      extractRecord(row, r);
+    } catch (...) {
+      goto skiprow;
```

Comment:
Yes, understood, like I said: 'strictly for alpha1'. I'm going to agree with the 'ignore them' policy for now, but we need to open a ticket to track this right after we merge it. Will wait for coverage of @cmouse's questions before merging.